R/arcadia_palette.R

```diff
@@ -216,19 +216,11 @@ BLUES <- list(colors = c(LAPIS, AEGEAN, ZEPHYR), positions = c(0, 0.254, 1.0))
 
 PURPLES <- list(colors = c(LILAC, ASTER, GHOST), positions = c(0, 0.144, 1.0))
 
-# Bicolor gradients
-# combine the gradients and inherit positions
-combine_gradients <- function(grad1, grad2) {
-  colors <- c(grad1$colors, rev(grad2$colors))
-  # Rescale the positions to fit the combined gradient
-  positions <- c(grad1$positions, rev(grad2$positions) + (1 - max(grad2$positions)))
-  list(colors = colors, positions = scales::rescale(positions))
-}
+ORANGE_SAGE <- list(colors = c(TERRACOTTA, TANGERINE, DAWN, LICHEN, SAGE, ASPARAGUS), positions = c(0, 0.3805, 0.5, 0.55, 0.8205, 1))
```

Comment:
This solution seems fine for now, but I would make sure that the lightest colors are symmetric about 0.5 to minimize any skew (assuming these particular values don't result in the exact same issue).

```suggestion
ORANGE_SAGE <- list(colors = c(TERRACOTTA, TANGERINE, DAWN, LICHEN, SAGE, ASPARAGUS), positions = c(0, 0.3805, 0.49, 0.51, 0.8205, 1))
```